manifests/a/Abyss/AbyssOverlay/1.2.2/Abyss.AbyssOverlay.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Abyss.AbyssOverlay
+PackageVersion: 1.2.2
+PackageLocale: en-US
+Publisher: Abyss

```

```suggestion
Publisher: Chit
Author: Abyss
```
Needs to match what control panel sees